cloudfront.cfhighlander.rb

```diff
@@ -8,6 +8,7 @@
 
     origins.each do |id,config|
       ComponentParam "#{id}OriginDomainName"
+      ComponentParam "#{id}OriginAccessIdentityInput"
```

Comment:
you should add a default value of ''

Reply:
maybe only add the OAI param if the origin is an S3 bucket

Reply:
Thanks @aaronwalker I've made these 2 changes